### DISCUSSION TITLE
Reset of Charger

### DISCUSSION THREAD

**Discussion by lieglw** [PatrikTrestik/homeassistant-solax-http#29] [Ideas]
Hey would it be possible to add the reset button. This helps when the charger goes into fault mode.
![image](https://github.com/PatrikTrestik/homeassistant-solax-http/assets/149818606/8bf9c6f9-6df0-4190-b7e8-95d8a65aad2c)

**Comment by PatrikTrestik** [comment]
I'll try. 
Only problem is to find mapping.
Give me few days to find hobby work time 😁

**Reply by lieglw** [reply]
Thank You 👍

**Comment by PatrikTrestik** [comment]
Added in v1.2.7
